packages/bits/src/docs/templates/partials/menu.hbs

```diff
@@ -1,487 +0,0 @@
-'use strict';
```

Comment:
Removed this because it's no longer needed after we reorganized the schematics docs. This change fixes https://jira.solarwinds.com/browse/NUI-5911